front/components/sparkle/navigation.tsx

```diff
@@ -38,6 +38,7 @@ export type SubNavigationAssistantsId =
   | "personal_assistants"
   | "data_sources_url"
   | "developers"
+  | "transcripts"
```

Comment:
I don't see this sub navigation even though the FF is enabled on my workspace.

Reply:
Old stuff - menu got moved from navigation. Removing.